tests/test_core/test_readers/test_read_soc.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from pathlib import Path
+import platform
+
+
+import spectrochempy as scp
+
+
+def test_read_SOC():
+    """upload and read surface oftics exemple"""
+
+    # the following does not work
+    baseurl = "https://github.com/chet-j-ski/SOC100_example_data/raw/main/"
```

Comment:
I am not sure If I have tested this, but I think this is not yet possible in read_remote. Anyway the best is to add the example files in spectrochempy_data.

Reply:
Yes, but I will ask the file owner first, and then add them in a next PR.